src/components/new-request/search.vue

```diff
@@ -566,5 +572,8 @@ export default class NewSearch extends Mixins(CommonMixin) {
   .v-messages:not(.error--text) {
     margin-bottom: -22px;
   }
+  .theme--light.v-list-item:not(.v-list-item--active):not(.v-list-item--disabled) {
+    color: #495057 !important;
```

Comment:
You can use a predefined theme colour opposed to the hex here, i believe this is `$gray7`.